services/backend/src/database/schema.prisma

```diff
@@ -190,28 +190,29 @@ model OpOfficeLocation {
 }
 
 model OpClassification {
-  id               String   @default(uuid()) @id
-  createdAt        DateTime @default(now())
-  updatedAt        DateTime @updatedAt
-  name             String   @unique
-  actingLevelUsers User[]   @relation("actingLevels")
-  groupLevelUsers  User[]   @relation("groupLevels")
+  id               String          @id @default(uuid())
+  createdAt        DateTime        @default(now())
+  updatedAt        DateTime        @updatedAt
+  name             String          @unique
+  actingLevelUsers User[]          @relation("actingLevels")
+  groupLevelUsers  User[]          @relation("groupLevels")
+  qualifiedPools   QualifiedPool[] @relation("classifications")
```

Comment:
The relation should not be classifications, since it would not be different than the other ones

```suggestion
  qualifiedPools   QualifiedPool[] @relation("qualifiedPools")
```